engine/src/main/java/org/terasology/engine/logic/players/DebugControlSystem.java

```diff
@@ -126,13 +118,6 @@ public void onKeyDown(KeyDownEvent event, EntityRef entity) {
         }
 
         switch (event.getKey().getId()) {
-            case Keyboard.KeyId.F11:
```

Comment:
Interestingly, the comment in the DebugProperties states that it is accessible via <kbd>F1</kbd> - probably just adds to the confusion, though...